app/config/AppConfig.scala

```diff
@@ -104,4 +105,20 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig) {
     (lang: String) => controllers.routes.LanguageSwitchController.switchToLanguage(lang)
 
   lazy val welshToggleEnabled: Boolean = servicesConfig.getBoolean("feature-switch.welshToggleEnabled")
+
+  //Enabled income sources
+  lazy val dividendsEnabled: Boolean = servicesConfig.getBoolean("feature-switch.dividendsEnabled")
+  lazy val interestEnabled: Boolean = servicesConfig.getBoolean("feature-switch.interestEnabled")
+  lazy val giftAidEnabled: Boolean = servicesConfig.getBoolean("feature-switch.giftAidEnabled")
+  lazy val employmentEnabled: Boolean = servicesConfig.getBoolean("feature-switch.employmentEnabled")
+
+  lazy val excludedIncomeSources: Seq[String] = {
```

Comment:
Is the plan that whenever we want to toggle these feature switches we would modify the configuration and restart the application? No runtime changes?

Reply:
For now, yeah